Library/Homebrew/cmd/audit.rb

```diff
@@ -548,11 +548,11 @@ def audit_github_repository
   end
 
   def audit_specs
-    if head_only?(formula) && formula.tap.to_s.downcase !~ /-head-only$/
+    if head_only?(formula) && formula.tap.to_s.downcase !~ /-?head-only$/
```

Comment:
Or make a single `if formula.tap` for both as you're checking it twice and then the relevant check for each.